.github/workflows/firebase-qa-pull-request.yml

```diff
@@ -46,5 +42,9 @@ jobs:
           expires: 7d
         env:
           FIREBASE_CLI_PREVIEWS: hostingchannels
+      - name: Deploy to Firebase
```

Comment:
Just out of curiosity, why the change in order on deployment?

Reply:
so we don't have to wait for the deployment to happen to get a preview link